lib/workers/repository/update/pr/code-owners.ts

```diff
@@ -21,37 +32,130 @@ export async function codeOwnersForPr(pr: Pr): Promise<string[]> {
 
     logger.debug(`Found CODEOWNERS file: ${codeOwnersFile}`);
 
+    // Get list of modified files in PR
     const prFiles = await getBranchFiles(pr.sourceBranch);
-    const rules = codeOwnersFile
-      .split(newlineRegex)
+
+    if (!prFiles) {
+      logger.debug('PR includes no files');
+      return [];
+    }
+
+    // Convert CODEOWNERS file into list of matching rules
+    const fileOwnerRules = codeOwnersFile
+      .toString()
```

Comment:
```suggestion
```
shouldn't necessary

Reply:
Solved in 8d6197b63848914cd42841eade789282bfca0efd